source/process/docs-reviewers-guide.rst

```diff
@@ -0,0 +1,116 @@
+Reviewer's Guide for docs.mattermost.com 
+========================================
+
+Review of pull requests to Mattermost documentation passes through the following high-level steps: 
+
+1. Project Members and Approved Contributors submit PRs to the docs.mattermost.com repository. 
+2. Incoming PRs are assessed and assigned review requirements (e.g. technical PRs, PRs requiring Sphinx changes, etc.). 
+3. Reviewers with appropriate qualifications either approve and merge the changes, or leave feedback for the submitter to being the review cycle again. 
+4. Once merged, PRs are verified by a Project Member and the submitter's work acknowledged. 
+5. Reviewers with consistent performance are acknowledge with an increase in skill level. Reviewers with repeated inconsistency may have their level decreased (though this is rare). 
```

Comment:
"acknowledge" should be "acknowledged".

Also this skill level thing is extremely confusing, you're referencing it before you've explained what it is.